.gitignore

```diff
@@ -1,9 +1,9 @@
+# npm
 node_modules/
 
 # Gatsby
 .cache/
-
-# Gatsby + production
+# + production
```

Comment:
Why is it "+ production"? Also I think `public` can just be under gatsby?

Reply:
Replaced with 
```
# Gatsby
.cache/
public/ 
```